miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MessageBridgeRatDispatcher.kt

```diff
@@ -50,6 +50,7 @@ internal class MessageBridgeRatDispatcher(private val miniAppAnalytics: MiniAppA
             ActionType.CLOSE_MINIAPP.action -> Actype.CLOSE_MINIAPP
             ActionType.GET_HOST_APP_THEME_COLORS.action -> Actype.GET_HOST_APP_THEME_COLORS
             ActionType.GET_IS_DARK_MODE.action -> Actype.GET_IS_DARK_MODE
+            ActionType.SEND_MA_ANALYTICS.action -> Actype.SEND_MA_ANALYTICS
```

Comment:
I guess, We might have to use MA_ANALYTICS_INFO in all the places as we already have the MAAnalytics.

Reply:
You are right @GondiTilak . I think as for this class, iOS uses the same keyword, so its ok. As for the rest of the class we have to use Info suffix